loopchain/baseservice/rest_client.py

```diff
@@ -206,5 +206,5 @@ def _create_jsonrpc_params(self, method: RestMethod, params: Optional[NamedTuple
             if "from_" in params:
                 params["from"] = params.pop("from_")
 
-        return Request(method.value.name, params) if params else Request(method.value.name)
+        return Request(method.value.name, **params) if params else Request(method.value.name)
```

Comment:
I wonder if this change does the same thing.
param match to `args` in `Request(method.value.name, params)`,
param match to `kwargs` in `Request(method.value.name, **params)`.

Reply:
Without unpacking params, it will causes unexpected situation.
AS `params` treated as one positional argument, `Request` makes `params` as a `list` (`{'params': [{'key': 'value'}]}`), not a desired `dict` type (`{'params': {'key': 'value'}}`)

Try to omit the commit `54a60e` in this branch and run test. (`$ pytest testcase/unittest/test_rest_client.py`)

See: https://github.com/bcb/jsonrpcclient/commit/38b90394311e27624b86f6e156b0b3e42e97f535

Reply:
how does work well in `jsonrpcclient 2.6.0`?

Reply:
@yakkle 
There have been many changes among the major update of JsonRPCClient.

2.6.0
https://github.com/bcb/jsonrpcclient/blob/2.6.0/jsonrpcclient/request.py#L99
If the parameter has only one dict, then it will be treated as keyword parameters.

3.4.0
https://github.com/bcb/jsonrpcclient/blob/3.3.4/jsonrpcclient/requests.py#L105
The codes have been removed.

Reply:
ok. It looks like using bug in `2.6.0`.